docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,242 @@
+   you are currently serving:
+    
+   ```text
+   no upgrades available (1.2.3 == 1.2.3)
+   an upgrade is available (1.2.3 -> 2.3.4)
+   ```
+    
+   `teleport-upgrade` may display warnings about not having a valid upgrade
+   schedule. This is expected immediately after install as the maintenance
+   schedule might not be exported yet.
+
+## Step 3/3. Enroll Kubernetes agents in automatic updates
+
+This section assumes that the name of your `teleport-kube-agent` release is
+`teleport-agent`, and that you have installed it in the `teleport` namespace.
+
+1. Confirm that you are using the Teleport Enterprise edition of the
+   `teleport-kube-agent` chart. You should see the following when you query your
+   `teleport-kube-agent` release:
+
+   ```code
+   $ helm -n `teleport` get values `teleport-agent` -o json | jq '.enterprise'
+   true
+   ```
+
+   If another value such as `null` is returned, update your existing agent
+   `values.yaml` to use the Enterprise version:
+
+   ```yaml
+   enterprise: true
+   ```
+
+1. Add the following chart values to the values file for the
+   `teleport-kube-agent` chart:
+
+   ```yaml
+   updater:
+     enabled: true
+   ```
+
+1. Update the Helm chart release with the new values:
+
+   ```code
+   $ helm -n <Var name="teleport" />  upgrade <Var name="teleport-agent" /> teleport/teleport-kube-agent \
+   --values=values.yaml \
+   --version=<Var name="(=cloud.version=)" />
+   ```
+
+1. You can validate the updater is running properly by checking if its pod is
+   ready:
+
+   ```code
+   $ kubectl -n teleport-agent get pods
+   NAME                               READY   STATUS    RESTARTS   AGE
+   <your-agent-release>-0                         1/1     Running   0          14m
+   <your-agent-release>-1                         1/1     Running   0          14m
+   <your-agent-release>-2                         1/1     Running   0          14m
+   <your-agent-release>-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
+   ```
+   
+1. Check for any deployment issues by checking the updater logs:
+   
+    ```code
+    $ kubectl -n <Var name="teleport" /> logs deployment/<Var name="teleport-agent" />-updater
+    2023-04-28T13:13:30Z	INFO	StatefulSet is already up-to-date, not updating.	{"controller": "statefulset", "controllerGroup": "apps", "controllerKind": "StatefulSet", "StatefulSet": {"name":"my-agent","namespace":"agent"}, "namespace": "agent", "name": "my-agent", "reconcileID": "10419f20-a4c9-45d4-a16f-406866b7fc05", "namespacedname": "agent/my-agent", "kind": "StatefulSet", "err": "no new version (current: \"v12.2.3\", next: \"v12.2.3\")"}
+    ```
+
+## Troubleshooting
+
+Teleport agents are not updated immediately when a new version of Teleport is
+released, and agent updates can lag behind the cluster by a few days.
+
+If the Teleport agent has not been automatically updating for several weeks, you
+can consult the updater logs to help troubleshoot the problem.
```

Comment:
Nit: Would be helpful to show how to "consult the logs". Maybe a `journalctl` command.